submissions/7SergSmith7/Friends-app/app.js

```diff
@@ -0,0 +1,143 @@
+import "../css/vendor/normalize.css";
+import "../css/styles.css";
```

Comment:
Why you importing CSS here and not in HTML? 🤔

Reply:
I use Webpack and it put styles in js file